k8s-install/README.md

```diff
@@ -1,9 +1,16 @@
 # Kubernetes self-hosted install
 
-Installation is simple:
+Installation is simple, for Kubernetes 1.5:
 
 ```
-kubectl apply -f canal.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/canal.yaml
+```
+
+For Kubernetes 1.6:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/canal.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/rbac.yaml
```

Comment:
to clear a few things up:

- delaying it after will not _delay_ it. The application will come up fine but will not have access to any of the resources in the kubernetes apiserver. This could be fine if retries are handled adequately. 
- It's split up because Service Accounts are used for all authorization, but the authorization method may be any number of methods, one of which is RBAC. So someone may decide they want to use ABAC instead of RBAC. By placing the RBAC roles in a separate file, our manifests are compatible with all authorization methods. (Whether or not this is practical / necessary is tbd. Maybe we recognize that noone is using ABAC and everyone has RBAC enabled, so we just want it in one file.)